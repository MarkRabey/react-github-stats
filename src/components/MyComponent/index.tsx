import * as React from 'react';

import styles from './MyComponent.module.scss';
export interface MyComponentProps {
  title?: string;
}

const MyComponent: React.FC<MyComponentProps> = ({title}) => {
  return (
    <div className={styles.container}>
      {title && (
        <header>
          <h1>MyComponent</h1>
        </header>
      )}

      <p>This is MyComponent</p>
    </div>
  );
};

export default MyComponent;
