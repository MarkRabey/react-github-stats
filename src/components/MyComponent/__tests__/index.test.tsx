import React from 'react';
import renderer from 'react-test-renderer';

import MyComponent from '..';

describe('MyComponent', () => {
  it('renders correctly', () => {
    const result = renderer.create(<MyComponent />).toJSON();
    expect(result).toMatchSnapshot();
  });
});
