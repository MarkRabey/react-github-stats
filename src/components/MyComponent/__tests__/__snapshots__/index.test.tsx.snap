// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent renders correctly 1`] = `
<div>
  <p>
    This is MyComponent
  </p>
</div>
`;
